packages/@n8n/task-runner-python/src/task_executor.py

```diff
@@ -45,10 +53,17 @@ def execute_process(
                 TaskExecutor.stop_process(process)
                 raise TaskTimeoutError(task_timeout)
 
-            returned = queue.get()
+            if process.exitcode != 0:
+                assert process.exitcode is not None
+                raise TaskProcessExitError(process.exitcode)
```

Comment:
Could we also include stderr/stdout on the error? Might help in debugging possible issues. Or are they logged automatically to console?